landd_intergration/README.md

```diff
@@ -0,0 +1,97 @@
+# OpenEdx Data Integration with L&D Services
+
+The Following Integration includes the transfer of course details and user's consumption data from OpenEdx to L&D services
+1. All the Meta Data of Courses created by staff in Openedx will be sent to L&D servies which shows up on Infopedia site
+2. User's entered in Infopedia courses will be redirected to OpenEdx site based on the Course URL
+3. Course Consumption data will be sent back to Infopedia SITE
+
+Here are the API's used: 
+
+**OpenEdx:**    
+
+  - Course catalog API         -- GET /api/courses/v1/courses/
+  - Bulk Gradees API           -- GET /api/grades/v1/user_grades/?username=all
+
+
```

Comment:
i'd consider describing the security acl'ing we're doing here (or we can have that conversation separately in OXA Microsoft Teams)